.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+	"recommendations": [
+        "ziglang.vscode-zig",
+        "ms-vscode.cpptools"
```

Comment:
huh, I always tried (and faild) to use [vscode-lldb](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb). But yeah, it does seem that sticking with gdb is better.

Reply:
Yes, I also tried lldb on Windows, but GDB just works.
I should have mentioned GDB's dependency in the docs.